storybookjs/storybook
# feat: add letterspacing and lineheight to Typeset component

Signed-off-by: André Jaenisch <andre.jaenisch@posteo.de>

Issue: #10560

## What I did

I've added lineHeight and letterSpacing prop to Typeset.

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

I've extended the Typeset story accordingly.



N.B.: Would there be value in typing these properties using `CSSProperties` of React types instead of string / number? This is likely out-of-scope for this PR, though.
